pkg/cmd/subcommand/install.go

```diff
@@ -13,24 +15,24 @@ import (
 	"syscall"
 	"time"
 
-	"k8s.io/apimachinery/pkg/util/version"
-
 	"github.com/OpenFunction/cli/pkg/client"
 	"github.com/OpenFunction/cli/pkg/cmd/util"
+	multispinner "github.com/OpenFunction/cli/pkg/cmd/util/spinner"
 	"github.com/OpenFunction/cli/pkg/components/common"
 	"github.com/OpenFunction/cli/pkg/components/inventory"
+	"github.com/oliveagle/jsonpath"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
-	"golang.org/x/sync/errgroup"
+	"k8s.io/apimachinery/pkg/util/version"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	k8s "k8s.io/client-go/kubernetes"
 )
 
-var (
-	w io.Writer
+const (
+	openFunctionLatestReleaseUrl = "https://api.github.com/repos/OpenFunction/OpenFunction/releases/latest"
```

Comment:
can be optimized for Chinese users with https://www.jsdelivr.com/

Reply:
yes, I have tested this before and it works